File: java/gpt/P6.java
// Task p6: Create vehicles array, prompt for index and print, then loop and print all
import java.util.Scanner;

public class P6 {
    public static void main(String[] args) {
        String[] vehicles = {"car", "truck", "motorcycle", "bicycle", "scooter"};
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter index of vehicle: ");
        try {
            int idx = Integer.parseInt(sc.nextLine().trim());
            if (idx >= 0 && idx < vehicles.length) {
                System.out.println("Selected: " + vehicles[idx]);
            } else {
                System.out.println("Index out of range");
            }
        } catch (NumberFormatException e) {
            System.out.println("Invalid input; please enter an integer");
        }

        System.out.println("All vehicles:");
        for (String v : vehicles) {
            System.out.println("- " + v);
        }
        sc.close();
    }
}

File: python/gpt/p6.py
# Task p6: Create vehicles array, prompt for index and print, then loop and print all


def main():
    vehicles = ["car", "truck", "motorcycle", "bicycle", "scooter"]
    try:
        idx = int(input("Enter index of vehicle: "))
        if 0 <= idx < len(vehicles):
            print("Selected:", vehicles[idx])
        else:
            print("Index out of range")
    except ValueError:
        print("Invalid input; please enter an integer")

    print("All vehicles:")
    for v in vehicles:
        print("-", v)


if __name__ == "__main__":
    main()
